tox.ini

```diff
@@ -1,125 +1,35 @@
 [tox]
-minversion = 3.5.0
-skipsdist = True
-skip_missing_interpreters = True
+minversion = 3.18.0
 isolated_build = True
-toxworkdir = {env:TOXWORKDIR:/tmp/{env:USER}/tox}
-envlist = lint
-          py3-agent
-          datalog
-          postprocess
-          tool-scripts
-          util-scripts
-          bench-scripts
-          py3-server
-          server
+toxworkdir = {env:TOXWORKDIR:/var/tmp/{env:USER}/tox}
+distshare = {toxworkdir}/distshare
+envlist = py
 
 [testenv]
-usedevelop = True
+description = Lints code and runs all agent and server unit/functional tests
 install_command = pip install --no-cache-dir --progress-bar off --prefix={envdir} {opts} {packages}
 passenv =
     PY_COLORS
     NO_COLORS
     TERM
+    HOME
```

Comment:
Unfortunately, that does not work because the way we have constructed the CI environment requires HOME to have a `.local` hierarchy with pbench installed in it, along with all of it's dependencies.

Reply:
Peter assures me that, in a subsequent PR, we're going to wean the CI off of `--current-env`, at which point we won't need `HOME` here, and so this won't be a problem.

Reply:
Tracked in #2681.